libsql-server/src/auth/user_auth_strategies/mod.rs

```diff
@@ -2,17 +2,26 @@ pub mod disabled;
 pub mod http_basic;
 pub mod jwt;
 
-use axum::http::HeaderValue;
+use anyhow::Context;
 pub use disabled::*;
 pub use http_basic::*;
 pub use jwt::*;
```

Comment:
Maybe this was missed in the last review but it would be good to remove the glob imports and make this more verbose. Can be done in a separate PR but I want to make sure we don't miss this.